openwisp-utils-qa-checks

```diff
@@ -82,6 +85,12 @@ runisort() {
     || { echoerr "ERROR: Isort check failed!"; FAILURE=1; }
 }
 
+runblack() {
+  black --check --diff . \
```

Comment:
add the `-S` flag because most modules use mainly single quotes, so we can avoid having to reformat all the files of each module which will help us when reviewing historic changes in the future

Reply:
@atb00ker :point_up: :blush:

Reply:
done